src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -240,10 +241,10 @@ def register_extra_cli_argument(command, dest, **kwargs):
     _cli_extra_argument_registry[command][dest] = CliCommandArgument(dest, **kwargs)
 
 def cli_command(module_name, name, operation,
```

Comment:
Should we add a **kwargs argument if we expect to add more of these "pass-through" parameters?